### DISCUSSION TITLE
New Design Request

### DISCUSSION THREAD

**Discussion by ilay01** [quasarframework/quasar#17197] [Ideas / Proposals]
Hey Guys, as you know Material Design is kinda outdated and many use Tailwind CSS or something similar! So please please update the look of Quasar so that it can stay with time

**Comment by pratikpatel-aurochs** [comment]
I've been using Quasar since version 0.17, and I appreciate its comprehensive component library and event handling. However, I'm now considering redesigning some components. Many developers are shifting to Tailwind CSS because it offers efficient and streamlined styling options. Although Quasar can achieve similar results, it often requires more time and effort.

What are your thoughts, @rstoenescu?

**Comment by 021-projects** [comment]
Agreed! I'm crazy about the functionality that Quasar offers, but most of the time I have to redesign the components to make them look more compact and modern. It would be awesome if the framework updated the components design <3
